src/sql/base/browser/ui/table/plugins/headerFilter.plugin.ts

```diff
@@ -86,10 +88,27 @@ export class HeaderFilter<T extends Slick.SlickData> {
 		const event = new StandardKeyboardEvent(e);
 		if (this.menu && event.keyCode === KeyCode.Escape) {
 			this.hideMenu();
+			if (this.previouslyFocusedElement?.focus && this.previouslyFocusedElement.tabIndex !== -1) {
+				this.previouslyFocusedElement?.focus();
+			}
 			EventHelper.stop(e, true);
 		}
 	}
 
+	private async handleGridKeyDown(e: KeyboardEvent): Promise<void> {
+		const event = new StandardKeyboardEvent(e);
+		if (event.keyCode === KeyCode.F3) {
```

Comment:
I'd put a comment here explaining why you're doing this since it's not really a common practice anywhere else AFAIK

Reply:
you mean add comment for why we are using F3 to open the filter menu?

Reply:
Yes - brief description of why this keybinding and why use a keybinding at all vs the standard tab + enter.